docs/src/main/asciidoc/rest.adoc

```diff
@@ -201,6 +201,10 @@ public class Endpoint {
 }
 ----
 
+Quarkus also extends the JAX-RS spec to allow you to use static methods. If the static method is on a class annotated with
+`@Path`, it will be treated the same as a non-static endpoint. If there is no `@Path` annotation on the class then the path
+is assumed to be a root path. This is especially useful for Kotlin as it allows you to define endpoints without creating a class.
+
```

Comment:
I'm not sure if we should add a Kotlin code sample here, but it sounds like it would be a good place to add one.